packages/nimble-components/src/breadcrumb-item/styles.ts

```diff
@@ -57,14 +57,12 @@ export const styles = css`
         text-decoration: underline;
     }
 
-    .start,
-    .end {
-        display: flex;
-        align-items: center;
+    [part='end'] {
```

Comment:
Is this right? Looks like `end` is styled twice but there are no styles for `start`.

Reply:
No, it's not right. I messed that up when I was cleaning up the code before posting my PR 🤦